src/test/java/io/kokuwa/maven/helm/AbstractMojoTest.java

```diff
@@ -127,7 +127,7 @@ static Server getServer(String id, String username, String password) {
 	}
 
 	static Path copyPackagedHelmChartToOutputdirectory(AbstractHelmMojo mojo) {
-		Path source = Paths.get("src/test/resources/app-0.1.0.tgz");
+		Path source = Paths.get("src/test/resources/__files/app-0.1.0.tgz");
```

Comment:
moved to new directory as WireMock expects files to live in a __files/ directory

Reply:
Is this the only location in the codebase that accesses the chart tarball?

Reply:
yup, verified with a comprehensive search